openbas-api/src/main/java/io/openbas/rest/mapper/form/ImportMapperUpdateInput.java

```diff
@@ -15,15 +15,15 @@
 public class ImportMapperUpdateInput {
 
```

Comment:
ImportMapperAdd and ImportMapperUpdate are the same?

Reply:
No, the importers property is not the same between the two. One use InjectImporterAddInput and the other InjectImporterUpdateInput. Those two objects are different